src/parser/jobs/brd/modules/RagingStrikes.tsx

```diff
@@ -56,7 +56,7 @@ class BarrageEvaluator extends ExpectedActionsEvaluator {
 	}
 
 	override countUsed(window: HistoryEntry<EvaluatedAction[]>, action: TrackedAction) {
-		if (action.action.id === this.barrageId) {
+		if (ensureArray(action.action)[0].id === this.barrageId) {
```

Comment:
i'm not keen on shovelling off this array->first operation onto every single consumer, especially given by and large they have no realistic reason to need to deal with it.

i think there's a few ways to go about reducing the impact of this change, the main ones i'd consider are
- updating `TrackedAction::action` to just be `TrackedActions::actions` and be done with it - this doesn't make it less impact, but it does make it much more predictable
- Using some generics and suchforth to allow using wider types of `TrackedAction` that incorporate extra info.